corehq/apps/hqadmin/service_checks.py

```diff
@@ -222,6 +223,42 @@ def check_formplayer():
         return ServiceStatus(res.ok, msg)
 
 
+def check_pillowtop():
+    def changes_behind(pillow_meta):
+        lag = 0
+        for k, offset in pillow_meta['offsets'].items():
+            # seq_format is 'json' for KafkaChangeFeed and 'text' for
+            # ChangeFeed. The former seems to have offsets that are greater
+            # than the corresponding seq, while the latter has a seq value
+            # that's greater than the offset
```

Comment:
minor readability suggestion: maybe consider abstracting this complex `seq` extraction business to a function.